client.go

```diff
@@ -59,10 +61,12 @@ func (e Error) Error() string {
 }
 
 // NewClient creates a client with the given authorization token.
-func NewClient(token string, options ...ClientOption) *Client {
+func NewClient(token string, edgeID string, edgeIPAddress string, options ...ClientOption) *Client {
 	client := &Client{
```

Comment:
nothing serious but shall we add ip check for egdeIPAddress since it's in IPV4 / IPV6 format? @takassh

Reply:
but again nothing serious. let's ignore